types/fundamental-react/lib/ActionBar.d.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+
+export type ActionBarProps = {
+    className?: string;
+    /* Set to **true** for mobile view of the Action Bar.*/
+    mobile?: boolean;
+    /* The width of the Action Bar in mobile view. */
+    width?: string;
+} & { [x: string]: any };
+
+export type ActionBarActionsProps = {
+    className?: string;
+} & { [x: string]: any };
+
+export type ActionBarBackProps = {
+    onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    className?: string;
+    buttonProps?: { [x: string]: any };
+} & { [x: string]: any };
+
+export type ActionBarHeaderProps = {
+    title: string;
+    className?: string;
+    /* Localized text for the description. */
+    description?: string;
+    /* Additional props to be spread to the description's `<p>` element. */
+    descriptionProps?: { [x: string]: any };
+    /* Heading level. `<h1>` is reserved for the page title. */
+    headingLevel?: 1 | 2 | 3 | 4 | 5 | 6;
+    titleProps?: { [x: string]: any };
+} & { [x: string]: any };
+
+export class ActionBar extends React.Component<ActionBarProps> {
```

Comment:
It looks like `fundamental-react` organizes its internal exports as `default` exports and then exports them in the index.js with different names. this code exports with the final name and then the index.d.ts just exports without renaming. There's no functional difference between the two, but if you wanted you could use default exports here like the original library.

Reply:
Thank you for your feedback. It turned out that it did make a difference in this project. I reworked the folder structure and covered all the other ways to import classes from fundamental-react.

Reply:
```
import { FormSet } from "fundamental-react";
import { FormItem } from "fundamental-react/lib";
import { FormLabel } from "fundamental-react/lib/Forms";
import FormInput from "fundamental-react/lib/Forms/FormInput";
```